nhs_context_logging/formatters.py

```diff
@@ -176,7 +192,7 @@ def _flatten_fields(
                 yield from self._flatten_fields(asdict(value), parent_key=key)
                 continue
 
-            if isinstance(value, (list, tuple)):
+            if isinstance(value, Iterable) and not isinstance(value, str):
```

Comment:
should this also be   `not isinstance(value, str, bytes)` ?  and I'm wonder wondering about  other dangerous Iterables ...  someome mistakenly does   log(result=stream)  ( and includes a 1GB file in their logs ) .

Reply:
yep makes sense, will put that in now